tidb node group support gcp (#20509)
pingcap/docs: tidb-cloud/tidb-node-group-management.md

## Prerequisites

- You have a [TiDB Cloud Dedicated](/tidb-cloud/select-cluster-tier.md#tidb-cloud-dedicated) cluster deployed on AWS or Google Cloud.
- You are in the **Organization Owner** or **Project Owner** role of your organization. For more information, see [User roles](/tidb-cloud/manage-user-access.md#user-roles).

> **Note**:
>

> > The TiDB Node Group can not be created during the cluster creation. You need to add the groups after the cluster is created and in Available state. 

Reviewer: ```suggestion
> The TiDB node groups cannot be created during the cluster creation. You need to add the groups after the cluster is created and in the **Available** state. 
```

> 

Reviewer 2: @qqqdan Should we include a link to the [Connect to a TiDB Cloud Dedicated Cluster via Google Cloud Private Service Connect](https://docs.pingcap.com/tidbcloud/set-up-private-endpoint-connections-on-google-cloud/) document in this step?
https://github.com/pingcap/docs/blob/7ac6502a7e7eb003451df5a06bf03fcc08963433/tidb-cloud/tidb-node-group-management.md?plain=1#L97

> > The TiDB node groups cannot be created during the cluster creation. You need to add the groups after the cluster is created and in the **Available** state. 

Reviewer: ```suggestion
> The TiDB node groups cannot be created during cluster creation. You need to add the groups after the cluster is created and in the **Available** state. 
```

2. In the upper-right corner, click **Connect**. A connection dialog is displayed.
3. Select your TiDB node group from the **TiDB Node Group** list and **Private Endpoint** from the **Connection Type** list.
4. In the left navigation pane, click **Networking**, and then select your TiDB node group from the **TiDB Node Group** list in the upper-right corner.

> 5. Click **Create Private Endpoint Connection** to create a new connection for this node group. For AWS clusters, please refer to [Connect to a TiDB Cloud Dedicated Cluster via Private Endpoint with AWS](/tidb-cloud/set-up-private-endpoint-connections.md). For Google Cloud clusters, please refer to [Connect to a TiDB Cloud Dedicated Cluster via Google Cloud Private Service Connect](https://docs.pingcap.com/tidbcloud/set-up-private-endpoint-connections-on-google-cloud/).

Reviewer 2: ```suggestion
5. Click **Create Private Endpoint Connection** to create a new connection for this node group. For clusters deployed on AWS, refer to [Connect to a TiDB Cloud Dedicated Cluster via Private Endpoint with AWS](/tidb-cloud/set-up-private-endpoint-connections.md). For clusters deployed on Google Cloud, refer to [Connect to a TiDB Cloud Dedicated Cluster via Google Cloud Private Service Connect](/tidb-cloud/set-up-private-endpoint-connections-on-google-cloud.md).
```
